src/components/TopBarMenu/Cart/Cart.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import AuthContext from "../../Quantity/Quantity";
+
+import "./Cart.scss";
+
+export default function Cart() {
+  const { quantity } = React.useContext(AuthContext);
+

```

Comment:
Now you need to grab again the context using useContext from React:
```suggestion
export default function Cart() {
  const { cart } = React.useContext(CartContext);
  console.log(cart) // { quantity: 0, setCart: ...  }
```